.bazelrc

```diff
@@ -0,0 +1,3 @@
+# Force the tests to not run in sandboxed mode, since it can introduce
+# errors and flakes.
+test --spawn_strategy=local
```

Comment:
Is it necessary for this file to be in the project's root directory?

Reply:
Yes, Bazel requires this